.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,93 @@
+name: Build and upload to PyPI
+
+# Build on every branch push, tag push, and pull request change:
+on: push
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 2
+      matrix:
+        os: [ubuntu-18.04, macos-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.6'
+
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel==1.4.2
+
+      - name: Cache OpenCV dependency for macOS
+        if: ${{ runner.os == 'macOS' }}
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-opencv
+        with:
+          path: |
+            /opt/opencv
+            ~/opencv
+            ~/opencv_contrib
+          # The key based on the file hash of the build script
+          # i.e. changing OPENCV_VERSION within the script will invalidate the cache
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('scripts/_install_opencv_*_on_macos.sh') }}
+
+      - name: Build wheels
+        env:
+          # FIXME (19-Jun-12020) Having permissions issues with docker hub
+          # CIBW_MANYLINUX_X86_64_IMAGE: wildme/manylinux:latest
+          CIBW_MANYLINUX_X86_64_IMAGE: mmulich/manylinux:latest
+          CIBW_BUILD: cp36-manylinux_x86_64 cp36-macosx_x86_64
+          # Script to prepare the macos environment
+          CIBW_BEFORE_BUILD_MACOS: bash scripts/ci_prepare_macos_for_build.sh
+        run: |
+          python -m cibuildwheel --output-dir wheelhouse
```

Comment:
Is there a reason we cannot use `cibuildwheel` on Ubuntu as well?

Reply:
It is being used on ubuntu. You don't see it directly, but a few lines up is the matrix definition

Reply:
Gotcha, I find these YAML files a bit hard to follow